package.json

```diff
@@ -31,7 +35,7 @@
   "devDependencies": {
     "@cypress/webpack-preprocessor": "~5.1.1",
     "@empathybroker/deep-merge": "^1.2.1",
-    "@empathy/eslint-plugin-x": "^1.2.0",
+    "@empathyco/eslint-plugin-x": "^1.2.0",
```

Comment:
Can you move this dependency below the `@empathy/x-translations` one? So it is ordered alphabetically ;)